contracts/src/access/access_control/extensions/enumerable.rs

```diff
@@ -48,35 +47,31 @@ pub const STORAGE_KEY: u32 = openbrush::storage_unique_key!(Members);
 
 #[derive(Default, Debug)]
 #[openbrush::upgradeable_storage(STORAGE_KEY)]
-pub struct Members {
+pub struct Data {
     pub role_members: MultiMapping<RoleType, AccountId, ValueGuard<RoleType>>,
```

Comment:
This has to be changed (in case of this extension there is a Mapping as well as a Mapping